insights/parsers/ps.py

```diff
@@ -384,8 +385,20 @@ def get_environ(self, proc):
         .. note::
            'proc' must contain the command
         """
-        env = []
+        ENV_RE_PATTEN_1 = r'[a-zA-Z_]+[a-zA-Z0-9_]*?=[\S]*'
```

Comment:
small typo: `PATTERN` instead of `PATTEN`.

Also, please add some comments that describe each of these patterns and your algorithm that uses them.